test/server/http/admin_test.cc

```diff
@@ -23,6 +23,7 @@
 #include "common/stats/thread_local_store.h"
 
 #include "server/http/admin.h"
+#include "server/http/admin_filter.h"
```

Comment:
That is needed actually, since the `AdminInstanceTest` class has a dependency on `AdminFilter` (see https://github.com/envoyproxy/envoy/blob/9186ebf374b1bd7d7c65ed1cec748377f88ec0a7/test/server/http/admin_test.cc#L655)